src/style.css

```diff
@@ -1 +1,116 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+
+  --lite-gray: #e1e7ed;
+  --wite: #fff;
+  --bg-color: #f6f6f7;
+  --text-color: #3D4E61;
+}
+
+.main {
+  padding: 12px 0;
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+}
+
+.search {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid var(--lite-gray);
+  background-color: var(--wite);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+}
+
+.search--big {
+  height: 70px;
+  padding-left: 25px;
+}
+
+.search--small {
+  height: 42px;
+  padding-left: 11px;
+}
+
+.search:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search:active {
```

Comment:
not `active`, but `:focus`
read about these states of an element: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus

Also, now it applies only to the small area on the left side
![Screenshot from 2023-12-07 20-09-23](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/0b8bccfb-8068-4d5f-b945-c64e25654252)

you should add gradient to input on focus